### ISSUE TITLE
Licensing, copyright and contributors

### ISSUE THREAD

**Issue by gocom** [textpattern/textpattern-branding#1]
This repository needs licensing, copyright and contributor information. What license do the assets use? Were do they originate and how many have contributed to them? What are the typefaces? How were they created? By whom and when?

We can not license the 3rd-party trademarked logos, since we don't own those, no matter how many hours you have put work into them.

**Comment by rwetzlmayr** [comment]
[Yet another discussion about logo usage guidelines](http://forum.textpattern.com/viewtopic.php?pid=295850#p295850).

Dean is silent so the trademark situation stays in limbo. 

But we can at least tuck a license on the logo and word mark. @belipe would [grant any license for his work](https://github.com/textpattern/textpattern-branding/issues/1#issuecomment-21857972): 

> From my side you may assign any license that suits best

@philwareham [prefers MIT](https://github.com/textpattern/textpattern-branding/issues/1#issuecomment-21857157).

MIT is fine for me.

@belipe, are you ready to make your choice?

**Comment by philwareham** [comment]
I'm not sure if Dean actually has any claim to the trademark now, since there has never been a clear trademark policy or trademark registration, and he doesn't continue to enforce (maybe never did) and or/defend the unregistered trademark, and isn't an active member of the project. Therefore the project itself has claim to the copyright (with the core devs as its overseers right now).

Just read this doc which sheds some light on it (albeit from 8 years ago): https://www.plagiarismtoday.com/2007/11/15/is-trademark-the-new-copyright/

The logo branding creative copyright is owned by @belipe with permission for us to use it - and hopefully with a licence if @belipe agrees (MIT being my choice of licence for this). Any website design copyright we have belongs to whoever created it, with their permissions for it to be used by the project, if we choose to enforce it.

Long story short: Put a licence on the branding. Put a trademark policy on the website (here's the WordPress one https://wordpress.org/about/domains/ - although bear in mind they have registered their trademark).

**Comment by belipe** [comment]
MIT is fine with me (I thought we already discussed this, didn’t we? As for the trademark this might really be not that easy (and costly by the way). And we’re not commercial like WordPress is, but we could still use their domain policy so everything directly related to the Textpattern community would go by a textpattern… domain and everything else should use txp… And, we could use the same policy for logo usage.

**Comment by philwareham** [comment]
@belipe an unregistered trademark (i.e. not paying for the registration) is still legally valid if it is stated and maintained. All we really need to do is add clear wording on our website.

'txp' is fair for others to use, although I would hope to discourage people from using any elements from the branding logotype to promote anything other than on the official project sites. The only reason we even need a stated licence for distribution is so that the branding repo can be pulled into other official Textpattern site repos.

Although that is your right to say yes or no to any usage enquiries, as the creative copyright holder of the logotype.

**Comment by belipe** [comment]
@philwareham I totally wouldn’t mind a bunch of fanboys and girls running around sticking the Textpattern logo to anything and everything if this made the best CMS in the world (still) more prominent even though it our brand would be corrupted a little bit ;-)

**Comment by philwareham** [comment]
Issue has become non-relevant, so closing. We (the development team) own the Textpattern brand now and a license has been stated.
